### PR TITLE
89 index graph

### PR DESCRIPTION
**Pull request by Cozloschi**

### CODE REVIEW

**Comment by vtemian** [comment]
Fix conflicts and will review this later.

**Comment by Cozloschi** [comment]
@vtemian

File: src/templates/dashboard/home.html [hunk 1]
```diff
@@ -51,16 +53,49 @@
...
+                        var color = $(data.element._node).css('stroke');
+                        if(colors.indexOf(color) < 0)
+                            colors.push(color);
+                    }
+                });
+
+                chart.on('created', function(){
+                    for(var i=0;i<=colors.length;i++){
+                        if(typeof colors[i] !== 'undefined'){
+                            $('.legend.sites').append('<span class="legend_item" ><a style="color:'+ colors[i] +'" href="/provider/'+provider_names[i]+'">'+website_names[i]+'</a></span>');
```
Comment (vtemian) [inline src/templates/dashboard/home.html]: line to long

File: src/templates/dashboard/home.html [hunk 2]
```diff
@@ -51,16 +53,49 @@
...
+                chart.on('draw', function (data) {
+                    if(data.type == 'point'){
+                        var color = $(data.element._node).css('stroke');
+                        if(colors.indexOf(color) < 0)
+                            colors.push(color);
+                    }
+                });
+
+                chart.on('created', function(){
+                    for(var i=0;i<=colors.length;i++){
```
Comment (vtemian) [inline src/templates/dashboard/home.html]: spaces after `;`

File: src/templates/dashboard/home.html [hunk 3]
```diff
@@ -51,16 +53,49 @@
             <h2>Why are we doing this?</h2>
             <p>We’re doing this so you can at last hack into the hosting and web performance jungle.</p>
         </div>
-	  </div>
-{% endblock %}
+        {% block chartist %}
+            <script type='text/javascript'>
+                websites = {{ top_websites|safe  }};
+                var times = [];
```
Comment (vtemian) [inline src/templates/dashboard/home.html]: Why do you need to create separate arrays for `time`, `value` etc.? 
It's easier to create a simple map and iterate over it.

**Comment by vtemian** [comment]
Merge conflicts

**Comment by Cozloschi** [comment]
@vtemian

File: src/templates/dashboard/home.html [hunk 4]
```diff
@@ -51,16 +53,55 @@
...
+                    if(data.type == 'point'){
+                        var color = $(data.element._node).css('stroke');
+                        if(colors.indexOf(color) < 0)
+                            data['colors'].push(color);
+                    }
+                });
+
+                chart.on('created', function(){
+                    forEach(data['colors'], function(key){
+                        $('.legend.sites').append(
```
Comment (vtemian) [inline src/templates/dashboard/home.html]:
```

Comment (Cozloschi) [inline src/templates/dashboard/home.html]: Line too long.
```
var link  = '<a style="color:{0}" href="/provider/{1}">{2}</a>'.format(
    data['colors'][key],
    data['provider_names'][key], 
    data['website_names'][key]
); ```

**Pull request merged**